tests/test_invalid.py

```diff
@@ -293,6 +293,14 @@ def test_invalid_clusters_num(self):
         with self.assertRaises(ValueError):
             MAB([0, 1], LearningPolicy.EpsilonGreedy(epsilon=0), NeighborhoodPolicy.Clusters(n_clusters=1))
 
+    def test_invalid_treebandit_lp_linucb(self):
```

Comment:
Should there also be a test to validate that if unsupported arguments for DecisionTreeClassifier's parameters are given it will raise an error? Or are we leaving that up to sklearn?

Reply:
_validate for TreeBandit does that I believe. Is that what you were referring to?

Reply:
Yes - checking that the keys are valid